src/js/lib/monster.util.js

```diff
@@ -1200,18 +1200,21 @@ define(function(require) {
 	}
 
 	/**
-	 * Returns the available features for a Kazoo phone number
+	 * Returns a list of features available for a Kazoo phone number.
 	 * @param  {Object} number  Phone number object, which contains the features details
 	 * @return {String[]}       Number's available features
 	 */
 	function getNumberFeatures(number) {
 		if (!_.isPlainObject(number)) {
 			throw new TypeError('"number" is not an object');
 		}
-		var numberFeatures = _.get(number, 'features_available', []);
-		return _.isEmpty(numberFeatures)
-			? _.get(number, '_read_only.features_available', [])
-			: numberFeatures;
+		var pathToFeatures = _.find([
+			'_read_only.features.available',
```

Comment:
Is this OK? The old path was `_read_only.features_available` (underscore instead of dot). Also, the order in which the paths are checked is inverted, because previously `features_available` was checked first.

Reply:
> Is this OK? The old path was `_read_only.features_available`

Yes, sorry if it wasn't clear but that's the whole point of this PR. The path was updated by https://github.com/2600hz/kazoo/pull/6291



> the order in which the paths are checked is inverted, because previously `features_available` was checked first.

That does not matter because:

- `features_available` is present when the number comes from `GET /phone_numbers`
- `_read_only.features.available` when it comes from `GET /phone_numbers/{PHONE_NUMBER}`